$NetBSD: patch-aa,v 1.4 2012/07/30 04:08:53 dholland Exp $

- use standard headers
- don't need to provide bcopy or strdup

--- lirec/li_recognizer.c.orig	Mon Jun 12 18:20:13 2000
+++ lirec/li_recognizer.c
@@ -21,7 +21,7 @@
 #include <stdio.h>
 #include <string.h>
 #ifndef ELX
-#include <malloc.h>
+#include <stdlib.h>
 #endif
 #include <math.h>
 #include <locale.h>
@@ -53,9 +53,9 @@ static char *lialg_recognize_stroke(rCla
 char* li_err_msg = NULL;
 char _zdebug_flag[128];
 
-#define bcopy(s1,s2,n) memcpy(s2,s1,n)
+/*#define bcopy(s1,s2,n) memcpy(s2,s1,n)*/
 
-#ifdef mips
+#if 0
 char *strdup(char* from) {
    char* to;
    int len = strlen(from) + 1;
@@ -923,7 +923,8 @@ RECOGNIZER_FINALIZE(r)
 ************************************************** */
 
 /*#include <assert.h>*/
-#include <values.h>
+#include <limits.h>
+#define MAXINT	INT_MAX
 #include <sys/time.h>
 
 #ifdef	__ultrix
